cli_tools_tests/go.mod

```diff
@@ -1,15 +1,20 @@
-module github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_integ_test/boot_inspect
+module github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_tests
```

Comment:
@zoran15 from #1379 

I thought e2e and module tests will be separate packages? We don't need to have a code dependency on cli_tools in E2E tests, only module tests need it. This dependency can potentially lead to e2e tests picking up this dependency as well (possibly by mistake).

Reply:
Looking back, `cli_tools` has been a dependency of the e2e tests since the original version of the package:

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/3c09ffd2f054e8f65153b22a8df712507ff1e7bc/cli_tools_e2e_test/go.mod

Given that, I'm curious about your concerns regarding the e2e tests having a dependency on `cli_tools`. What are some issues that you're thinking of?

Reply:
The original dependency to cli_tools was when OVF import e2e tests tested the importer instead of the actual binary. Now, this dependency is used for some utils that are used for testing, but not for actual tests. As these are CLI tools e2e tests, ideally, we would be testing CLIs (binaries/gcloud), and not depend on the code in cli_tools.

Reply:
Talked offline -- there are existing dependencies on cli_tools from e2e, and we decided to leave that unchanged for this PR